hw2/05-form.html

```diff
@@ -7,14 +7,81 @@
     <title>05 - Form</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="05-form.js"></script>
   </head>
-  <body>
-    <main>
-      <form>
-        <h1>Exercise 05 - Form</h1>
+  <body class="bg-dark">
+    <main class="container mt-5">
+      <form id="signup" class="bg-light border rounded p-4">
+        <h1 class="text-center">Exercise 05 - Form</h1>
+        <div class="mt-3">
+          <label class="fw-bold" for="name">Full Name:</label>
+          <br />
+          <input class="col-12" type="text" id="name" name="name" required />
+        </div>
+        <div class="mt-3">
+          <label class="fw-bold" for="email">Email:</label>
+          <br />
+          <input class="col-12" type="text" id="email" name="email" required />
+        </div>
+        <div class="mt-3">
+          <label for="registration" class="form-label fw-bold"
+            >Registration Status:</label
+          >
+          <select class="form-select" id="registration" name="registration">
+            <option selected disabled>Choose an Option</option>
+            <option value="registered">Registered</option>
+            <option value="notRegistered">Registered</option>
+          </select>
+        </div>
+        <section class="mt-3">
+          <label class="fw-bold"
+            >Have You Taken Any of the Following Courses?</label
+          >
+          <fieldset>
+            <div class="form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="programmingLanguages"
+              />
+              <label>Programming Languages</label>
+            </div>
+            <div class="form-check">
+              <input
+                class="form-check-input"
+                type="checkbox"
+                id="operatingSystems"
+              />
+              <label>Operating Systems</label>
+            </div>
+            <div class="form-check">
+              <input class="form-check-input" type="checkbox" id="fullstack" />
+              <label>Full Stack Web Development</label>
+            </div>
+          </fieldset>
+        </section>
+        <div class="mt-3">
+          <label for="additionalInfo" class="fw-bold">Anything Else?</label>
+          <input
+            type="text"
+            class="form-control"
+            id="additionalInfo"
+            name="additionalInfo"
+          />
```

Comment:
Use <textarea class="form-control" id="additionalInfo"></textarea> instead of <input /> this should solve your spacing issue